test/FunctionalTests/Tests/DataOData/Tests/OData.Common.Tests/PublicApi/PublicApi.bsl

```diff
@@ -2627,6 +2627,7 @@ public class Microsoft.OData.Edm.EdmIncludeAnnotations : IEdmIncludeAnnotations
 
 public class Microsoft.OData.Edm.EdmModel : Microsoft.OData.Edm.EdmModelBase, IEdmElement, IEdmModel {
 	public EdmModel ()
+	public EdmModel (bool enableBuiltInVocabularyModel)

```

Comment:
>enableBuiltInVocabularyModel [](start = 23, length = 28)

Maybe call "includeDefaultVocabularies"?

Reply:
Resolved.

---
In reply to: [248413340](https://github.com/OData/odata.net/pull/1367#discussion_r248413340) [](ancestors = 248413340)